.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
```

Comment:
Do you check every single variable for null or undefined before using it? No. Because sometimes it would be a programming error (not a runtime error). Likewise, in some cases `default:` is not needed in `switch` because only a fixed set of values can occur.